src/lib/geo-location.js

```diff
@@ -40,10 +40,11 @@ const mockLocation = {
  */
 const getGeoLocation = async ({ geoCountry, mode, offline, state }) => {
   const cacheObject = state.get(STATE_GEO_PROPERTY)
+  if (geoCountry) mode = 'mock'
```

Comment:
I think adding a comment here would help.

```suggestion

  // If `--country` was used, we also set `--mode=mock`. 
  if (geoCountry) {
    mode = 'mock'
  }
```